### DISCUSSION TITLE
Custom Function Request

### DISCUSSION THREAD

**Discussion by coolman888** [coinfoundry/miningcore#1050] [General]
Dear sir,
I want to add the following functions to miningcore for ETH:

1. 24h shares statistics with Valid/Stale/Invalid
2. Reported Hashrate submit by mining software with "eth_submitHashrate” params.
3. Offline workers.
I will pay for these functions, Please let me kwon the quote. Drop me an email to jaychew888@gmail.com
Thanks.
Jay

**Comment by PMPMining** [comment]
any luck on this ?
